src/appshell/qml/Preferences/AppearancePreferencesPage.qml

```diff
@@ -35,6 +35,7 @@ PreferencesPage {
         id: appearanceModel
     }
 
+
```

Comment:
Unnecessary change

Reply:
Again, a result of my experiments and me forgetting to clean up afterwards. :(
Fixed.